app/static/src/scss/poc-search.scss

```diff
@@ -1,16 +1,17 @@
 /* CSS for Proof of concept Search Page */
 
-.govuk-form-group {
+.govuk-form-group--poc-search {
+
   width: 53rem;
 }
 
-.govuk-button {
+.govuk-button--poc-search {
   margin-top: -4.3rem;
```

Comment:
Just as a warning - negative margins _can_ cause layout issues. I wonder if there is an easier way of positioning these elements?

Reply:
(We agreed to have a separate discussion about this)